client/galaxy/scripts/mvc/ui/ui-select-content.js

```diff
@@ -283,8 +278,8 @@ var View = Backbone.View.extend({
 
     /** Change of type */
     _changeType: function() {
-        let Galaxy = getGalaxyInstance();
-        var self = this;
+        let self = this;
+        let galaxy = getGalaxyInstance();
```

Comment:
We should pick a standard and follow it here.  The rest of the codebase uses `Galaxy` to refer to the singleton object.  I suppose an argument could be made that it should be lowercased since it's not a constructor, but I'm kinda partial to `Galaxy` myself.  Anyone else have feelings on this?

Reply:
I like that it can be lower-case now indicating a local variable. Gives some peace of mind.

Reply:
Well that's the rub, right?  It's a local reference to a singleton, there's one Galaxy object for the entire app.  I do slightly prefer `Galaxy` to indicate that it's not just a throwaway local object, but we should decide this as a group and stick with it instead of just changing one at a time, here and there.

Reply:
I agree, I noticed that we recently started to switch to the lower-case those I followed up with it on any new code. Eitherway is fine with me. Once we have some consensus we should stick to it.